src/DockWidget.cpp

```diff
@@ -516,12 +516,36 @@ bool CDockWidget::event(QEvent *e)
 		{
 			d->ToggleViewAction->setText(title);
 		}
+		if (d->DockArea)
+		{
+			d->DockArea->updateTitleBarVisibility();//update tabs menu

```

Comment:
The comment indicates that the intention of this call is to update the tabs menu - is this right? In this case I would prefer if you directly mark the tabs menu outdated by calling the markTabsMenuOutdated() function of the DockAreaTitleBar